frontend/ui/src/components/Note/index.tsx

```diff
@@ -14,29 +14,33 @@ type NoteProps = {
 };
 
 const Note = ({note, onClick, removeNote, updateNote}: NoteProps): JSX.Element => {
-  return (
-    <div className={styles.note} onClick={onClick}>
-      <div className={`${styles.noteInner} ${onClick ? styles.clickable : ''} ${removeNote ? styles.isRemovable : ''}`}>
-        <div>
-          <span className={`${styles.noteNameExpanded}`}>{note.text}</span>
-          {removeNote && (
-            <span className={styles.removeNote} onClick={removeNote}>
-              <Close className={styles.closeButton} title="Delete" />
+  if (note.timestamp && note.text) {
+    return (
+      <div className={styles.note} onClick={onClick}>
+        <div
+          className={`${styles.noteInner} ${onClick ? styles.clickable : ''} ${removeNote ? styles.isRemovable : ''}`}
+        >
+          <div>
+            <span className={`${styles.noteNameExpanded}`}>{note.text}</span>
+            {removeNote && (
+              <span className={styles.removeNote} onClick={removeNote}>
+                <Close className={styles.closeButton} title="Delete" />
+              </span>
+            )}
+            <span className={styles.removeNote} onClick={updateNote}>
+              <EditPencilIcon className={`${styles.editNote}`} title="Edit note" />
             </span>
-          )}
-          <span className={styles.removeNote} onClick={updateNote}>
-            <EditPencilIcon className={`${styles.editNote}`} title="Edit note" />
-          </span>
-        </div>
-        <div>
-          <p className={`${styles.noteDate}`}>
-            {note.timestamp.toLocaleDateString()}{' '}
-            {note.timestamp.toLocaleTimeString([], {hour: '2-digit', minute: '2-digit'})}
-          </p>
+          </div>
+          <div>
+            <p className={`${styles.noteDate}`}>
+              {note.timestamp.toLocaleDateString()}{' '}
```

Comment:
We already have a generic [date formatter](https://github.com/airyhq/airy/blob/1ed5f94a556da6dae8b6811a7190df126f4ac548/frontend/ui/src/services/format/date.ts#L3). Let's use that instead :)